client/src/legacy/TabSet.js

```diff
@@ -46,22 +56,35 @@ $.entwine('ss', function($){
       });
     },
 
+    /**
+     * @func triggerLazyLoad
+     * @desc Find all the lazy loadable gridfield and nodes in the panel and trigger their reload.
+     * @param {Object} panel
+     */
+    triggerLazyLoad: function(panel) {
+      panel.find('.grid-field--lazy-loadable, .lazy-loadable').each((i, el) => {
+        const $el = e(el);
+        // Avoid triggering all lazy loadable scripts when using nested tabs
+        if ($el.closest('.ss-tabset, .cms-tabset').is(this)) {
+          // Trigger dedicated entwine function (see GridField.js)
+          if(typeof $el.lazyload === 'function') {
+            $el.lazyload();
```

Comment:
I'm not sure where that `e` variable comes from but I assume it's meant to be the jQuery var?
We don't tend to prefix any variables in javascript with `$`, so either rename `$el` to something else or just remove that declaration.
```suggestion
        const element = $(el);
        // Avoid triggering all lazy loadable scripts when using nested tabs
        if (element.closest('.ss-tabset, .cms-tabset').is(this)) {
          // Trigger dedicated entwine function (see GridField.js)
          if(typeof element.lazyload === 'function') {
            element.lazyload();
```

Reply:
yes sorry, that's because I was using the uncompiled source and made a copy paste and forgot to change back the jquery selector... i'm implementing your suggestions